### PR TITLE
Simple planning simulator update for latest develop

### PR DESCRIPTION
**Pull request by TakaHoribe**
## Description

```
X　c3cb428ff (main) Change formatter to clang-format and black (#2332)
O　7832179d3 Feature/add ideal accel model interface (#1894) ⇨　すでにautoで考慮されてた
X　8a9af0afe add sort-package-xml hook in pre-commit (#1881)
考慮済み　883951fb3 Add autoware api (#1979)
X　58b6a7cd5 fix some typos (#1941)
X　13cd1dd2c Fix -Wunused-parameter (#1836)
O　59bb6be98 add cov pub in psim (#1732)
X　485f3ac49 Add markdownlint and prettier (#1661)
X　6d255448e Add pre-commit (#1560)
X　9f6a2ba3e Fix lint errors (#1378)
O　336cd4145 Refactor vehicle info util (#1305)
```

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: simulator/simple_planning_simulator/launch/simple_planning_simulator.launch.py [hunk 1]
```diff
@@ -57,6 +57,7 @@ def generate_launch_description():
             ('input/trajectory', '/planning/scenario_planning/trajectory'),
             ('output/twist', '/vehicle/status/velocity_status'),
             ('output/odometry', '/localization/kinematic_state'),
+            ('output/cov', '/localization/pose_with_covariance'),
```
Comment (rej55) [inline simulator/simple_planning_simulator/launch/simple_planning_simulator.launch.py:60]: @TakaHoribe 
lane_departure_checkerではodometryから共分散を作っているので、pose_with_covarianceはodometryに含めます？
https://github.com/tier4/AutowareArchitectureProposal.iv/blob/8dadf31b11846c8132d443b6daf7e9748e0451f8/control/lane_departure_checker/src/lane_departure_checker_node/lane_departure_checker.cpp#L136

Comment (TakaHoribe) [reply simulator/simple_planning_simulator/launch/simple_planning_simulator.launch.py:60]: あ、確かにそうですね！修正しておきます！

Comment (TakaHoribe) [reply simulator/simple_planning_simulator/launch/simple_planning_simulator.launch.py]: Done in https://github.com/tier4/AutowareArchitectureProposal.iv/pull/735/commits/2c52f0e5bcf52207d350145427361e33c3563471

**Review by rej55 — APPROVED** [review]
LGTM

**Pull request merged**